aiidalab_widgets_base/computational_resources.py

```diff
@@ -1071,6 +1069,9 @@ def on_setup_code(self, _=None):
 
             kwargs = {key: getattr(self, key).value for key in items_to_configure}
 
+            # convert computer to AiiDA node from its UUID
```

Comment:
I don't see the conversion to UUID here?

Reply:
Well, the `kwargs["computer"]` is a UUID of a computer. I admit this is confuse, maybe it is clear to name the key as "computer_uuid" then convert and pass the computer instance to the `InstalledCode`. 

All confusing from `orm.InstalledCode(**kwargs)`, I think it is better to make this clear by convert and passing computer as parameter explicitly.

Reply:
addressed.